fala/apps/adviser/utils.py

```diff
@@ -0,0 +1,46 @@
+# utils.py
+
+CATEGORY_TRANSLATIONS = {
+    "aap": "claims-against-public-authorities",
+    "med": "clinical-negligence",
+    "com": "community-care",
+    "crm": "crime",
+    "deb": "debt",
+    "disc": "discrimination",
+    "edu": "education",
+    "mat": "family",
+    "fmed": "family-mediation",
+    "hou": "housing",
+    "hlpas": "hlpas",
+    "immas": "immigration-asylum",
+    "mhe": "mental-health",
+    "mosl": "modern-slavery",
+    "pl": "prison-law",
+    "pub": "public-law",
+    "wb": "welfare-benefits",
+}
+
+SLUG_TO_CODE = {v: k for k, v in CATEGORY_TRANSLATIONS.items()}
+
+
+def get_category_code_from_slug(slug):
+    return SLUG_TO_CODE.get(slug)
+
+
+def get_category_display_name(category_code):
+    return CATEGORY_TRANSLATIONS.get(category_code)
+
+
+CATEGORY_MESSAGES = {
```

Comment:
Again on this this python file into it's own. Something like `constants/category_messages.py`

This doesn't need to be a class, just something that can be isolated into the app that's using it.